data/Psi/basicPsi4.0/water_mp2_fdgrad.in

```diff
@@ -0,0 +1,12 @@
+molecule water {
```

Comment:
I think this should go in our regression suite (https://github.com/cclib/cclib-data). Let us know if you need help organizing that.

Reply:
Are you suggesting regression suite because it's another MP2 water example? There's precedence for different examples run with different settings (e.g. ADF), so I think it could stay.

Reply:
Yes, and I think we should move the ADF examples with slightly different settings to regressions as well. I believe we should keep the unit test suite as compact as possible. Otherwise we'll eventually end up with multiple variants for all basic job types.

Reply:
I was assuming regression tests were more for fringe cases and bugs, whereas this is more of a core feature. I can move it to regressions. Alternatively, what do you think of just adding a line to the original water_mp2_in `findif_grad = gradient('mp2', dertype=0)` and updating the output file?

Reply:
No, there is a lot of "core feature"-type testing in our regressions, because of the sometimes vastly different capabilities of packages.

In this specific case, I propose a specific `fdgrad` and eventually `fdhess` test. I know I wrote those tests for Q-Chem, and it's an important use-case that packages will only calculate finite-difference gradients and Hessians for correlated methods.

Reply:
I've now removed the unit test and instead created a regression test. Output file and the addition of the path in regressionfile.txt are in a separate PR on the cclib-data repo right now  cclib/cclib-data#77. Let me know if I need to change anything. Thanks for the help and suggestions.

Reply:
@berquist we can add this case as a unit test later, but I would prefer to do so for at least 3-4 packages at once so that we established some reasonable coverage.